pkg/dialect/types.go

```diff
@@ -0,0 +1,77 @@
+package dialect
+
+import (
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+	"github.com/gophersbd/ormpb/protobuf"
+)
+
+// Supported Data type
+const (
+	Int       = "INT"
```

Comment:
are those types dialect dependent? may be also move inside dialect?

Reply:
These are dialect independent. kept in types for easy readability